exotica/include/exotica/MotionSolver.h

```diff
@@ -51,6 +51,7 @@ namespace exotica
         virtual void InstantiateBase(const Initializer& init);
         virtual void specifyProblem(PlanningProblem_ptr pointer);
         virtual void Solve(Eigen::MatrixXd & solution) = 0;
+        virtual void Solve(const std::vector<Eigen::VectorXd>& init, Eigen::MatrixXd & solution);
```

Comment:
Can you move the ```std::vector<Eigen::VectorXd> init``` into the ```UnconstrainedTimeIndexedProblem``` and let AICO pick it up from there. The solve method can then stay simple and you can specify your init trajectory inside the problem.

Reply:
Yes, it will break the current AICO API though.

Reply:
Looking at it closer it will require more changes to AICO and we would need to figure out which way we want it in the future:

a) the initial guess is part of the problem and to be set there - requires updating AICO and then iLevator
b) the initial guess is passed in as an argument, similar to how it's done in AICO and iLeavtor